src/neptune/versionHelper.d

```diff
@@ -0,0 +1,334 @@
+/*******************************************************************************
+
+    Helper to deal with version strings
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module neptune.versionHelper;
```

Comment:
Yes, https://github.com/sociomantic-tsunami/neptune/blob/master/src/semver/Version.d

I don't want to distract @mathias-baumann-sociomantic with such trivialities until main functionality is done :)